src/drivers/transponder/sagetech_mxs/sg_sdk/toInt16.c

```diff
@@ -15,6 +15,10 @@
  */
 int16_t toInt16(const uint8_t bytes[])
 {
+	if (sizeof(bytes) / sizeof(int16_t) < 2) {
```

Comment:
```warning: ‘sizeof’ on array function parameter ‘bytes’ will return size of ‘const uint8_t*’```

`const uint8_t bytes[]` is the same as `const uint8_t* bytes`. `sizeof` does neither give you the total size of all elements nor the total amount of elements in the array.

Reply:
change revert